src/main/java/io/kestra/plugin/gcp/bigquery/AbstractJobInterface.java

```diff
@@ -46,9 +46,9 @@ public interface AbstractJobInterface {
     Map<String, String> getLabels();
 
     @Schema(
-        title = "Sets whether the job has to be dry run or no.",
-        description = " A valid query will return a mostly empty response with some processing statistics, " +
-            "while an invalid query will return the same error it would if it wasn't a dry run."
+        title = "Whether the job has to be dry run or not.",
+        description = " A valid query will mostly return an empty response with some processing statistics, " +
+            "while an invalid query will return the same error as it would if it were an actual run."
```

Comment:
```suggestion
            "while an invalid query will return the same error as it would if it was an actual run."
```

Reply:
"were" is correct in this context.
![Image 2023-12-30 at 3 09 05 PM](https://github.com/kestra-io/plugin-gcp/assets/6594483/b70e2a22-0b7a-4417-b3d4-126aa95ff84a)

Reply:
I learnt something :+1:

Reply:
I learnt something :+1: